pkg/api/validation/validation_test.go

```diff
@@ -2892,20 +2892,20 @@ func TestValidateLimitRange(t *testing.T) {
 			{
 				Type: api.LimitTypePod,
 				Max: api.ResourceList{
-					api.ResourceCPU:    resource.MustParse("100"),
-					api.ResourceMemory: resource.MustParse("10000"),
+					api.ResourceCPU:    resource.MustParse("100m"),
+					api.ResourceMemory: resource.MustParse("10000Mi"),
```

Comment:
What about test a large number like '1ZiB' to see whether it will exceed MaxInt64?